docs/getting_started/faq.md

```diff
@@ -135,6 +135,111 @@ all the properties of scaled units and is consistent with the rest of the librar
+
+    inline constexpr struct currency : quantity_spec<dim_currency> {} currency;
+
+    inline constexpr struct us_dollar : named_unit<"USD", kind_of<currency>> {} us_dollar;
+    inline constexpr struct scaled_us_dollar : named_unit<"USD_s", mag_power<10, -8> * us_dollar> {} scaled_us_dollar;
+
+    namespace unit_symbols {
+
+    inline constexpr auto USD = us_dollar;
+    inline constexpr auto USD_s = scaled_us_dollar;
+
+    }  // namespace unit_symbols
+
+    using Price = quantity<currency[us_dollar], double>;
+    using Scaled = quantity<currency[scaled_us_dollar], std::int64_t>;
+
+    Price price1 = 12.95 * USD;
+    Price price2 = 12.55 * USD;
+    Scaled spx1 = price1;
+    Scaled spx2 = price2;
+    ```
+
+=== "C++20"
+
+    ```cpp
+    inline constexpr struct dim_currency : base_dimension<"$"> {} dim_currency;
+
+    inline constexpr struct currency : quantity_spec<currency, dim_currency> {} currency;
+
+    inline constexpr struct us_dollar : named_unit<"USD", kind_of<currency>> {} us_dollar;
+    inline constexpr struct scaled_us_dollar : named_unit<"USD_s", mag_power<10, -8> * us_dollar> {} scaled_us_dollar;
+
+    namespace unit_symbols {
+
+    inline constexpr auto USD = us_dollar;
+    inline constexpr auto USD_s = scaled_us_dollar;
+
+    }  // namespace unit_symbols
+
+    using Price = quantity<currency[us_dollar], double>;
+    using Scaled = quantity<currency[scaled_us_dollar], std::int64_t>;
+
+    Price price1 = 12.95 * USD;
+    Price price2 = 12.55 * USD;
+    Scaled spx1 = price1;
+    Scaled spx2 = price2;
+    ```
+
+=== "Portable"
+
+    ```cpp
+    inline constexpr struct dim_currency : base_dimension<"$"> {} dim_currency;
+
+    QUANTITY_SPEC(currency, dim_currency);
+
+    inline constexpr struct us_dollar : named_unit<"USD", kind_of<currency>> {} us_dollar;
+    inline constexpr struct scaled_us_dollar : named_unit<"USD_s", mag_power<10, -8> * us_dollar> {} scaled_us_dollar;
+
+    namespace unit_symbols {
+
+    inline constexpr auto USD = us_dollar;
+    inline constexpr auto USD_s = scaled_us_dollar;
+
+    }  // namespace unit_symbols
+
+    using Price = quantity<currency[us_dollar], double>;
+    using Scaled = quantity<currency[scaled_us_dollar], std::int64_t>;
+
+    Price price1 = 12.95 * USD;
+    Price price2 = 12.55 * USD;
+    Scaled spx1 = price1;
+    Scaled spx2 = price2;
+    ```
+
+
+The above does not compile because of a truncating conversion from `double` to `std::int64_t`.
+To silence the error, a user might be tempted to do the following:
+
+```cpp
+Scaled spx1 = value_cast<std::int64_t>(price1);
+Scaled spx2 = value_cast<std::int64_t>(price2);
+```
+
+which will compile but provide an incorrect truncated value as the representation type conversion will
+happen before the value will be expanded/scaled to a target unit.
+
+To prevent such issues and make similar conversions safe by construction, we've added the following:
+
+```cpp
+Scaled spx1 = value_cast<USD_s, std::int64_t>(price1);
```

Comment:
The trend to require explicit units indeed results in safer code that doesn't unexpectedly change meaning over time.
However, consider the inverse, where `price1` is represented with `std::int64_t`, and you're casting to `double`.
You probably want the representation conversion first, and then the scaling for the unit.

Reply:
Maybe I worded this wrongly in FAQ, but I think it will also work with the current implementation. I have to confirm that.

Reply:
Confirmed!

```cpp
std::cout << value_cast<si::second, double>(42 * us) << "\n";
```

prints:

```text
4.2e-05 s
```

Reply:
Is the representation conversion happening first?
What about something more extreme like `value_cast<si::nanosecond, double>(42 * Ts)`?

Reply:
> The trend to require explicit units indeed results in safer code that doesn't unexpectedly change meaning over time.

Please note that it is not to future-proof the cast but to make the conversion correct in the first place. Without it, the value was truncated, and the result was invalid.

Reply:
> What about something more extreme like `value_cast<si::nanosecond, double>(42 * Ts)`?

```cpp
std::cout << value_cast<si::nano<si::second>, double>(42 * Ts) << "\n";
```

prints:

```text
4.2e+22 ns
```